src/pugbot/commands/mod.rs

```diff
@@ -22,3 +33,57 @@ pub fn error_embed(description: &'static str) -> Embed {
     video: None,
   }
 }
+
+// These groups really beg to be defined within the same module as the
+// commands they are grouping.
```

Comment:
👍 Yeah this code organization seems out of date now

Reply:
I'd plan on reshuffling the modules closer to the 0.10.x branch being merge-ready. There might be other factors to consider that come to light as we step through the remaining updates.